backend/src/main/java/gov/cdc/usds/simplereport/api/model/accountrequest/AccountRequest.java

```diff
@@ -26,9 +26,8 @@
   @JsonProperty @NotNull private String state;
   @JsonProperty @NotNull private String zip;
   @JsonProperty @NotNull private String county;
-  @JsonProperty @NotNull private String facilityType;
-  @JsonProperty private String facilityTypeOther;
```

Comment:
I would consider leaving these two fields so requests from the old UI can still be handled during the transition and removing them later.  Probability of that happening is probably small, so alternatively, we should just carefully watch for errors on `/account-request` for a little while after deploying.